client/result/result.go

```diff
@@ -0,0 +1,352 @@
+// Copyright (c) 2016 Uber Technologies, Inc.
```

Comment:
That makes sense. Essentially the same as current state with `client/result` ->`bootstrap/result`